deploy.sh

```diff
@@ -1,34 +1,17 @@
 #! /bin/bash
 
-# ensure REPO_URL is set externally (e.g. via CI/CD)
-if [ "x$REPO_URL" = "x" ]; then
-  echo "error: REPO_URL is required to be set"
-  exit 1
-fi
+# Show script in output, and error if anything fails
+set -xe
 
-echo "Docker deploy started"
-# 0. authenticate your Docker client to your registry:
-eval $(~/.local/bin/aws ecr get-login --no-include-email)
-
-# 0.5. set the current version:
 VERSION=$(node -p "require('./package.json').version")
-IMAGE_NAME="dashevo/dapi"
+IMAGE_NAME="dashpay/dapi"
 
```

Comment:
Let’s compare package version with the current build tag

Reply:
I did the same here as with Drive -- didn't want to change deploy logic, just to change the place where it's deployed to.

Reply:
I see, no problem. We will do that is a separate PR.